tck/src/main/java/org/eclipse/microprofile/openapi/tck/PetStoreAppTest.java

```diff
@@ -303,4 +305,19 @@ public void testAPIResponseSchemaDefaultResponseCode(String type) {
                         hasEntry(equalTo("type"), equalTo("object")),
                         hasEntry(equalTo("properties"), notNullValue())));
     }
+
+    @RunAsClient
+    @Test(dataProvider = "formatProvider")
+    public void testExtensionPlacement(String type) {
+        ValidatableResponse vr = callEndpoint(type);
+
+        String opPath = "paths.'/store/inventory'.get";
+
+        vr.body(opPath, hasEntry(equalTo("x-operation-ext"), equalTo("test-operation-ext")));
+
+        vr.body(opPath + ".responses.'200'", not(hasProperty("'x-operation-ext")));
+        vr.body(opPath + ".responses.'200'", hasEntry(equalTo("x-response-ext"), equalTo("test-response-ext")));
+        vr.body(opPath + ".responses.'500'", not(hasProperty("'x-operation-ext")));
+        vr.body(opPath + ".responses.'503'", hasEntry(equalTo("x-operation-ext"), equalTo("test-operation-ext")));
```

Comment:
Should these checks use `hasKey` rather than `hasProperty`?

Reply:
I'll try that, it looks better. The test did pass on my branch in smallrye-open-api with these changes.